test/unit/migrate_assets_to_asset_manager_test.rb

```diff
@@ -117,6 +117,13 @@ class AssetFilePathsTest < ActiveSupport::TestCase
     assert_equal 2, subject.file_paths.size
   end
 
+  test '#files includes hidden files' do
+    hidden_path = File.join(organisation_logo_dir, '.hidden.jpg')
+    FileUtils.cp(dummy_asset_path, hidden_path)
+
+    assert_equal 2, @subject.file_paths.size
```

Comment:
Minor: This assertion seems quite vague and not easy to understand at first glance. Would it not be better to check for the inclusion of the hidden file's path? I see that some of the existing tests take a similar approach, so I guess it would be worth changing them as well...

Reply:
Good point thanks! I've addressed this in b5d5770.